airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java

```diff
@@ -96,6 +104,40 @@ public MssqlSource() {
     super(DRIVER_CLASS, AdaptiveStreamingQueryConfig::new, new MssqlSourceOperations());
   }
 
+  @Override
+  public ConnectorSpecification spec() throws Exception {
+    if (cloudDeploymentMode()) {
+      final ConnectorSpecification cloudDeploymentSpec = Jsons.clone(super.spec());
+      // Remove "unencrypted" value for "ssl_method".
+      ((ArrayNode) cloudDeploymentSpec.getConnectionSpecification().get("properties").get(SSL_METHOD).get("oneOf")).remove(0);
```

Comment:
We do want to leave it then, no?
since check is going to verify that at least one mode of encryption is configured